thermal-control-design.md

```diff
@@ -198,21 +396,53 @@ We don't have a CLI for fan status getting yet, new CLI for fan status could be
 	  -?, -h, --help  Show this message and exit.
 
 	Commands:
-	  fanstatus  Show fan status information
+	  fan        Show fan status information
 	  mlnx       Mellanox platform specific configuration...
 	  psustatus  Show PSU status information
 	  summary    Show hardware platform information
 	  syseeprom  Show system EEPROM information
 The output of the command is like below:
 
-	admin@sonic# show platform fanstatus
-	FAN    Speed      Direction      Timestamp
-	-----  ---------  ---------  -----------------
-	FAN 1  12919 RPM  Intake     20191112 09:38:16
-	FAN 2  13043 RPM  Exhaust    20191112 09:38:16
+	admin@sonic# show platform fan
+	FAN    Speed      Direction  Presence     Status  Timestamp
+	-----  ---------  ---------  -----------  ------  -----------------
+	FAN 1  85%        Intake     Present      OK      20191112 09:38:16
+	FAN 2  60%        Exhaust    Not Present  Not OK  20191112 09:38:16
```

Comment:
we shall update the show command output to reflect fan drawer info.  and also DB schema need to be changed to have fan drawer info, right?